### PR TITLE
Updated lambda types

### PR DESCRIPTION
**Pull request by lucacasonato**
I had to manually copy these types from https://unpkg.com/@types/aws-lambda@8.10.59/index.d.ts because it is now multi file. I only had to make minor changes:
1. concat all into one file
2. remove all imports
3. remove callback type
4. remove callback parameter from handler type

cc @hayd

### CODE REVIEW

File: runtime/types [hunk 1]
```diff
@@ -1,27 +0,0 @@
-#!/bin/sh
-set -euo pipefail
-
-#
-# Update the version and run to update ./types.d.ts
-#
-
-# See latest version: https://www.npmjs.com/package/@types/aws-lambda
-# Note: DefinitelyTyped is MIT License.
-curl -fSsL https://unpkg.com/@types/aws-lambda@8.10.39/index.d.ts -o types.d.ts
```
Comment (hayd) [inline runtime/types:10]: So, updating here no longer worked?

Comment (lucacasonato) [reply runtime/types:10]: No, it is not a single file anymore, rather a folder full of files that reference each other.

**Comment by hayd** [comment]
Is it possible to script this (1-4)?

I am concerned about maintaining unversioned/manual copy-pasted code 😄 and want to make future updates easy... (though it didn't help with this update now it is no longer a single file!)

File: runtime/mod.ts [hunk 2]
```diff
@@ -1,120 +1,191 @@
...
-  ClientContextEnv,
-  CloudFormationCustomResourceCreateEvent,
-  CloudFormationCustomResourceDeleteEvent,
+  PrincipalValue,
+  MaybeStatementPrincipal,
+  MaybeStatementResource,
+  StatementAction,
+  StatementResource,
+  StatementPrincipal,
+  APIGatewayProxyHandler,
```
Comment (hayd) [inline runtime/mod.ts:52]: There is something strange about this ordering... it should be alphabetical?

**Comment by lucacasonato** [comment]
Added a script for automatic generation - a lot of regex, but it works.

**Pull request merged**